src/instances/fnctr/augmented.js

```diff
@@ -1,16 +1,14 @@
-const { _ } = require("@masaeedu/infix");
-
 const { implement } = require("../../plumbing");
+const ClassDef = require("../classdef");
 const { Functor, Foldable, Traversable } = require("../../classes");
 
 const Fn = require("../fn");
 const Arr = require("../arr");
 const Fnctr = require(".");
 const { Recurse } = Fnctr;
 
-const classes = [Functor, Foldable, Traversable];
-const derive = _(Fn)(classes)
-  ["|>"](Arr.map(implement))
-  ["|>"](Fn.pipe)._;
+const classes = Arr.fold(ClassDef)([Functor, Foldable, Traversable]);
 
-module.exports = { Fnctr: { ...Fnctr, Recurse: Ts => derive(Recurse(Ts)) } };
+module.exports = {
+  Fnctr: { ...Fnctr, Recurse: Ts => implement(classes)(Recurse(Ts)) }
```

Comment:
Not that this is a mega performant library anyway, but does calling `implement(classes)` outside of this lambda save anything?

Reply:
Not really, the only work `implement` can do before receiving the 2nd argument is computing the keys of `classes`.